sdk/java/src/main/java/com/github/onsdigital/dis/redirect/api/sdk/model/Redirect.java

```diff
@@ -0,0 +1,33 @@
+package com.github.onsdigital.dis.redirect.api.sdk.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * The model of a redirect as provided by the redirect API.
+ */
+public class Redirect {
+    /** The original path to redirect from. */
+    private @Setter @Getter String from;
+
+    /** The destination path to redirect to. */
+    private @Setter @Getter String to;
```

Comment:
```suggestion
    /** The destination path to redirect to. */
    @Setter @Getter
    private String to;
```
Would usually format like this (I think)